doc/source/whatsnew/v2.0.0.rst

```diff
@@ -179,6 +179,10 @@ the appropriate rows. Also, when ``n`` is larger than the group, no rows instead
 
     gb.nth(n=3, dropna="any")
 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In previous versions, an error occurred if the :meth:`dates` method is ran for a date range.
```

Comment:
You can just add this to one of the bug fixes sections below

Reply:
Ah I see I'll work on it now

Reply:
I've added this to the Timezones section.